bears/python/PyImportSortBear.py

```diff
@@ -145,6 +175,9 @@ def run(self, filename, file,
             dependencies that occur in many projects.
         :param max_line_length:
             Maximum number of characters for a line.
+        :param treat_seperated_imports_independently:
+            Treat import statements seperated by more then one line as
```

Comment:
then -> than

"..seperated by more then one line.." -- this sounds like two blank lines needed between import groups?? that would be a pep8 violation, and isnt what is in your tests.

I think "..separated by a blank line.." is better.